app/javascript/gobierto_dashboards/webapp/containers/shared/Nav.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <nav class="dashboards-home-nav">
+    <ul>
+      <router-link
+        :to="{ name: 'summary' }"
+        :class="{ 'is-active': activeTab === 0 }"
+        tag="li"
+        class="dashboards-home-nav--tab"
+      >
+        <i class="fas fa-chart-bar" />
+        <i class="far fa-chart-bar" />
+        <span>{{ labelSummary }}</span>
+      </router-link>
+      <router-link
+        :to="{ name: 'contracts_index' }"
+        :class="{ 'is-active': activeTab === 1 }"
+        tag="li"
+        class="dashboards-home-nav--tab"
+      >
+        <i class="fas fa-clone" />
+        <i class="far fa-clone" />
+        <span>{{ labelContracts }}</span>
+      </router-link>
+      <router-link
+        :to="{ name: 'tenders_index' }"
+        :class="{ 'is-active': activeTab === 2 }"
+        tag="li"
+        class="dashboards-home-nav--tab"
+      >
+        <i class="fas fa-clone" />
+        <i class="far fa-clone" />
+        <span>{{ labelTenders }}</span>
+      </router-link>
+    </ul>
+  </nav>
+</template>
+
+<script>
+export default {
+  name: 'Nav',
+  props: {
+    activeTab: {
+      type: Number,
+      default: 0
+    }
+  },
+  data() {
+    return {
+      labelSummary: I18n.t("gobierto_dashboards.dashboards.contracts.summary"),
+      labelContracts: I18n.t("gobierto_dashboards.dashboards.contracts.contracts"),
+      labelTenders: I18n.t("gobierto_dashboards.dashboards.contracts.tenders")
+    }
+  },
+  routesToNavBarMapping: {
+    'summary': 0,
+    'contracts_index': 1,
+    'contracts_show': 1,
+    'tenders_index': 2,
+    'tenders_show': 2
+  },
```

Comment:
I did it this way because I think the router may not need to know anything about the presentation (those are only the tab index for the Nav component).